examples/index.md

```diff
@@ -275,7 +275,7 @@ Content-Type: application/vnd.api+json
 ```
 
 > Note: in the responses above with a 422 status code, `400 Bad Request` would
-also be acceptable. ([http://stackoverflow.com/a/20215807/1261879](More details.))
+also be acceptable. [More details.](http://stackoverflow.com/a/20215807/1261879)
```

Comment:
weird, that should have rendered correctly. I guess [(More details.)](http://stackoverflow.com/a/20215807/1261879)` was what I wanted.